src/main/resources/anniv6Resources/localization/eng/GremlinCamp/Powerstrings.json

```
@@ -0,0 +1,7 @@
+{
+  "${ModID}:PlayerPoisonPower": {
+    "DESCRIPTIONS": [
+      "At the end of your turn, take #b%s damage, then reduce this by #b1."
```

If  you want to be consistent with the base game wording.
```suggestion
      "At the end of your turn, take #b%s damage, then reduce #yPoison by #b1."
```